### PR TITLE
⏫upgrade kustomize

### PR DESCRIPTION
**Pull request by magicsong**

### CODE REVIEW

File: config/samples/service.yaml [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-  template:
-    metadata:
-      labels:
-        app: mylbapp
-    spec:
-      containers:
-      - name: name
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
```
Comment (haslersn) [inline config/samples/service.yaml:32]: Why was this file removed?

Comment (duanjiong) [reply config/samples/service.yaml:32]: This is supposed to be just an example to explain how to use it, but if you want to know how to use it, you can refer to the documentation at the bottom of the doc directory.

Comment (haslersn) [reply config/samples/service.yaml:32]: The README refers to this file which (on master) doesn't exist anymore.

Comment (duanjiong) [reply config/samples/service.yaml:32]: ok， i see,  thanks for the reminder
